cmd/monaco/runner/runner.go

```diff
@@ -96,6 +98,11 @@ Examples:
 			}
 
 			memory.SetDefaultLimit()
+
+			additionalHeaders := environment.GetAdditionalHttpHeadersFromEnv()
+			if len(additionalHeaders) > 0 {
+				log.Info("Additional custom HTTP headers: %q, from '%s' environment variable", maps.Keys(additionalHeaders), environment.AdditionalHttpHeaders)
```

Comment:
We could increase the informational content of this log, maybe something like:
```suggestion
				log.Info("Additional headers %q from '%s' environment variable will be added to HTTP requests", maps.Keys(additionalHeaders), environment.AdditionalHttpHeaders)
```

Reply:
Good point, copied your suggestion